src/test/osd/ceph_test_osd_stale_read.cc

```diff
@@ -36,7 +36,7 @@ int get_primary_osd(Rados& rados, const string& pool_name,
     + oid
     + string("\",\"format\": \"json\"}");
   bufferlist outbl;
-  int r = rados.mon_command(cmd, inbl, &outbl, NULL);
+  [[maybe_unused]] int r = rados.mon_command(cmd, inbl, &outbl, nullptr);
   assert(r >= 0);
```

Comment:
The alternative is to switch to `ceph_assert()`.
I'm not sure now but weren't our `assert`s converted into `ceph_assert`s in the past?

Reply:
probably, we can just
```c++
if (int r = rados.mon_command(cmd, inbl, &outbl, NULL);
    r < 0) {
  return r;
}
```

Reply:
probably, we can just
```c++
if (int r = rados.mon_command(cmd, inbl, &outbl, NULL);
    r < 0) {
  return r;
}
```

Reply:
Reworked.